setup/nuvfile.yml

```diff
@@ -40,6 +40,8 @@ tasks:
           {{.RUN}} nuv setup nuvolaris login
           config STATUS_LAST=devcluster
       fi
+    - task: notify 
```

Comment:
The notification on a devcluster could result in spam as it could be a very common operation. @msciabarra what do you think?